src/engine/SConscript

```diff
@@ -3,16 +3,16 @@ import daos_build
 
 def scons():
     """Execute build"""
-    Import('env', 'prereqs')
+    Import('env', 'prereqs', 'libdaos_tgts')
 
     denv = env.Clone()
 
     denv.Append(CPPDEFINES=['-DDAOS_PMEM_BUILD'])
-    libraries = ['daos_common_pmem', 'gurt', 'cart', 'vos_srv', 'daos']
+    libraries = ['daos_common_pmem', 'gurt', 'cart', 'vos_srv']
     libraries += ['bio', 'dl', 'uuid', 'pthread', 'abt']
-    libraries += ['hwloc', 'pmemobj', 'protobuf-c']
+    libraries += ['hwloc', 'pmemobj', 'protobuf-c', 'isal']
```

Comment:
(pylint-import-outside-toplevel) Import outside toplevel (prereq_tools.PreReqComponent)

Reply:
Is this warning telling us the line number mutation code is incorrect on this 1.2 branch?  I do remember we did have to fix it recently.

Reply:
Yes, there are 3 issues I ran into here that are fixed on master branch

1. warnings in generated wrapper code 
2. bandit
3. The NLT one that I cherry picked in this patch.